Utility/ARM/Update-AzureModule.ps1

```diff
@@ -28,8 +40,36 @@ Param
     [String] $AutomationResourceGroup,
 
     [Parameter(Mandatory=$True)]
-    [String] $AutomationAccount
+    [String] $AutomationAccount,
+
+    [Parameter(Mandatory=$False)]
+    [object] $ModuleVersionOverrides
     )
+
+$versionOverrides = ""
+## Try to parse module version overrides
+if ($ModuleVersionOverrides){
+    if ($ModuleVersionOverrides.GetType() -eq (@{}).GetType()){
```

Comment:
Consider "-eq [hashtable]". The result is the same, but looks a bit more readable.